app/javascript/layouts/sidebar/LayoutSidebar__Root.res

```diff
@@ -37,12 +39,12 @@ let userDetails = currentUserName => {
           <p className="text-base font-medium text-gray-700 group-hover:text-gray-900">
             {s(currentUserName)}
           </p>
-          <p className="text-sm font-medium text-gray-500 group-hover:text-gray-700">
+          <button className="text-sm font-medium text-gray-500 group-hover:text-gray-700">
```

Comment:
Is this optimized for mobile view also ?

Reply:
Yeah 🎉. As it was. 📍